test/Generator/ValueGeneratorTest.php

```diff
@@ -498,4 +497,55 @@ public function testExceptionInvalidValue(mixed $value, string $type): void
         $this->expectExceptionMessage('Type "' . $type . '" is unknown or cannot be used');
         $valueGenerator->generate();
     }
+
+    /**
+     * @param ValueGenerator::OUTPUT_* $outputMode
+     */
+    #[DataProvider('multipleOutputArray')]
+    public function testArrayWithOutputMode(
+        array $array,
+        string $type,
+        string $outputMode,
+        string $output
+    ): void {
+        $valueGenerator = new ValueGenerator($array, $type, $outputMode);
+
+        self::assertSame($valueGenerator->generate(), $output);
+    }
+
+    /**
+     * Data provider for testArrayWithOutputMode test
+     */
+    public static function multipleOutputArray(): array
```

Comment:
This error is expected: PHPUnit 10 is not yet understood by `psalm/plugin-phpunit`.

Re-generating the baseline should fix this.

Reply:
I'm not sure what to do with this information, can I do something there?

Reply:
@thomas-kl1 
See: https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#using-a-baseline-file